Sources/Clappr/Classes/Base/Container.swift

```diff
@@ -63,6 +65,15 @@ open class Container: UIObject {
         }
     }
 
+    @objc open func reload(source: String) {
+        let mimeType = options[kMimeType] as? String
+
+        options[kDefaultSubtitle] = playback?.selectedSubtitle?.language
```

Comment:
What do you think about separating this set of default options in another function?

Reply:
Done